packages/esm-patient-chart-app/src/visit/hooks/useVisitAttributeType.tsx

```diff
@@ -0,0 +1,86 @@
+import { FetchResponse, openmrsFetch, showNotification } from '@openmrs/esm-framework';
+import { useMemo } from 'react';
+import useSWR from 'swr';
+
+interface VisitAttributeType {
+  uuid: string;
+  display: 'Insurance Type';
```

Comment:
Are we sure that this will always be the display value? I think just making it `string` here would be sufficient and potentially closer to the truth.